web-ui/src/pages/FeedbackSubmitPage.jsx

```diff
@@ -38,6 +39,17 @@ const FeedbackSubmitPage = () => {
 
   
   useEffect(() => {
+    if (!requestQuery || requestQuery===undefined) {
```

Comment:
Checking `requestQuery === undefined` after checking `!requestQuery` is redundant. If `requestQuery` is undefined, the `!requestQuery` will evaluate to false and short-circuit the conditional. Thus there is no case where the logic reaches the check for `requestQuery === undefined`; it can be safely removed.